shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -877,13 +877,96 @@ async fn test_protocol_v5(#[case] _driver: CassandraDriver) {
     .start()
     .await;
 
-    // let connection = || {
-    //     CassandraConnectionBuilder::new("127.0.0.1", 9042, driver)
-    //         .with_protocol_version(ProtocolVersion::V5)
-    //         .build()
-    // };
+    let connection = || {
+        CassandraConnectionBuilder::new("127.0.0.1", 9042, driver)
+            .with_protocol_version(ProtocolVersion::V5)
+            .build()
+    };
 
-    // standard_test_suite(&connection, driver).await;
+    standard_test_suite(&connection, driver).await;
 
     shotover.shutdown_and_then_consume_events(&[]).await;
 }
+
+#[rstest]
+#[case::cdrs(CdrsTokio)]
```

Comment:
Can we do something like `#[case::cdrs_v5(CdrsTokio, V5)` to avoid duplicating this test?

Reply:
I'll actually just remove this test and leave the pass through ones for now. Need to come up with a better testing plan for v5, once the compression stuff is done that may become more clear.

Reply:
sounds good, lets land this